pyproject.toml

```diff
@@ -32,5 +32,4 @@ serve-docs = "mkdocs serve"
 members = ["rye-devtools"]
 
 [tool.ruff]
-# the .rye folder is added by the rye github action
-exclude = [".rye", ".venv"]
+exclude = [".venv"]
```

Comment:
Is this needed or can we pull this out here too? Doesn't Ruff have a set of defaults it excludes (include .venv)?

Reply:
Yeah. `.venv` is ignored by default.

Reply:
Pushed another commit to remove it too